src/command/DefaultMenus.js

```diff
@@ -253,11 +253,11 @@ define(function (require, exports, module) {
         // editor_cmenu.addMenuItem(Commands.NAVIGATE_JUMPTO_DEFINITION);
         editor_cmenu.addMenuItem(Commands.TOGGLE_QUICK_EDIT);
         editor_cmenu.addMenuItem(Commands.TOGGLE_QUICK_DOCS);
-        editor_cmenu.addMenuDivider();
-        editor_cmenu.addMenuItem(Commands.EDIT_CUT);
-        editor_cmenu.addMenuItem(Commands.EDIT_COPY);
-        editor_cmenu.addMenuItem(Commands.EDIT_PASTE);
-        editor_cmenu.addMenuDivider();
+		editor_cmenu.addMenuDivider();
```

Comment:
Please use the correct indentation, and please make sure to run `npm test` before committing code, because `npm test` will inform you of these easy to catch errors. They should not show up in a pushed commit